crates/router/src/core/payouts.rs

```diff
@@ -182,7 +278,7 @@ pub async fn payouts_create_core(
     .await?;
 
     // Form connector data
-    let connector_data = get_connector_data(
+    let connector_details = get_connector_choice(
```

Comment:
We will need to call `get_connector_choice` for PayoutsUpdate flow as well. In case connector could not be decided during PayoutsCreate